certif/translations/en.json

```
@@ -131,22 +132,35 @@
       },
       "import": {
         "title": "Créer/éditer plusieurs sessions",
-        "cancel": "Annuler l'import",
-        "session-import-template": "Télécharger (.csv)",
-        "session-import-template-dl-error": "Une erreur s'est produite pendant le téléchargement",
-        "session-import-template-label": "Télécharger le modèle vierge",
-        "session-import-upload": "Importer (.csv)",
-        "session-import-upload-label": "Importer le modèle complété",
-        "instructions-creation-title": "Pour créer des sessions ?",
-        "instructions-creation-list": "Vous pouvez créer des sessions :",
-        "instructions-creation-list-with-candidates": "<strong>Avec candidats</strong>, complétez le modèle dans son intégralité,",
-        "instructions-creation-list-without-candidates": "<strong>Sans candidat</strong>, complétez uniquement les informations des sessions.",
-        "instructions-edition-title": "Pour éditer la liste des candidats de sessions déjà créées ?",
-        "instructions-edition-replace": "<strong>Pour remplacer</strong> une liste pré-existante dans le fichier modèle,",
-        "instructions-edition-subscription": "<strong>Pour inscrire</strong> des candidats dans une session vide.",
-        "instructions-edition-warning": "Attention à renseigner le numéro de session sans les autres informations de sessions (ex: date, heure, etc.) pour éviter les doublons lors de la création.",
-        "breadcrumb-import": "Import du modèle",
-        "breadcrumb-summary": "Récapitulatif"
+        "previous-page-button": "Revenir à la liste des sessions",
+        "step-one": {
+          "cancel": "Annuler l'import",
+          "session-import-template": "Télécharger (.csv)",
+          "session-import-template-dl-error": "Une erreur s'est produite pendant le téléchargement",
+          "session-import-template-label": "Télécharger le modèle vierge",
+          "session-import-upload": "Importer (.csv)",
+          "session-import-upload-label": "Importer le modèle complété",
```

Si on suit l'ADR, on pourrait partir sur qq chose comme : 
```suggestion
          "actions": {
                    "cancel": "Annuler l'import",
                    "session-import-template": {
                              "title": "Télécharger (.csv)",
                              "error": "Une erreur s'est produite pendant le téléchargement",         
                              "extra-information": "Télécharger le modèle vierge",
                    }
                    "session-import-upload": {
                              "title": "Importer (.csv)",
                              "extra-information": "Importer le modèle complété,
                    }
          },
```